src/main/resources/au/com/centrumsystems/hudson/plugin/buildpipeline/extension/BuildCardExtension/buildCardHelpers.jelly

```diff
@@ -7,17 +7,5 @@
 
         By default we just include helpers to append script tags to your build card
     -->
-    <![CDATA[
 
-    <script type="text/javascript">
-        Handlebars.registerHelper('startScript', function() {
-            return new Handlebars.SafeString("<scr" + "ipt>");
-        });
-
-        Handlebars.registerHelper('endScript', function() {
-            return new Handlebars.SafeString("</scr" + "ipt>");
-        });
-    </script>
```

Comment:
Should not be a non-draft PR if it is not mergable, and this is not mergable. Should extract the existing content into a new script, remove the CDATA section, and clean up the string concatenation.

Reply:
@basil 
I think i misunderstood what needs to be done for this

Did you mean instead to do something like the below instead of removing completely?

Handlebars.registerHelper('startScript', function() {
    return new Handlebars.SafeString("<script>");